--- a/configure
+++ b/configure
@@ -15861,9 +15861,4 @@
           *)
-                                                                                    if xcrun --sdk macosx10.5 --show-sdk-path >/dev/null 2>&1 ; then
-                 mac_version_min="-mmacosx-version-min=10.5"
-                 mac_sysroot="-isysroot $(xcrun --sdk macosx10.5 --show-sdk-path)"
-              else
-                 mac_version_min="-mmacosx-version-min=10.6"
-                 mac_sysroot="-isysroot $(xcrun --sdk macosx --show-sdk-path)"
-              fi
+                 mac_version_min=""
+                 mac_sysroot="-isysroot $(xcrun --show-sdk-path)"
           esac
--- a/src/common/pa_converters.c
+++ b/src/common/pa_converters.c
@@ -1418,6 +1418,7 @@
 {
     PaInt16 *src = (PaInt16*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void)dest;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1458,6 +1459,8 @@
 {
     PaInt16 *src = (PaInt16*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
