--- a/configure
+++ b/configure
@@ -15861,9 +15861,4 @@
           *)
-                                                                                    if xcrun --sdk macosx10.5 --show-sdk-path >/dev/null 2>&1 ; then
-                 mac_version_min="-mmacosx-version-min=10.5"
-                 mac_sysroot="-isysroot $(xcrun --sdk macosx10.5 --show-sdk-path)"
-              else
-                 mac_version_min="-mmacosx-version-min=10.6"
-                 mac_sysroot="-isysroot $(xcrun --sdk macosx --show-sdk-path)"
-              fi
+                 mac_version_min=""
+                 mac_sysroot="-isysroot $(xcrun --show-sdk-path)"
           esac
--- a/src/common/pa_converters.c
+++ b/src/common/pa_converters.c
@@ -455,6 +455,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     PaInt32 temp;
 
     (void) ditherGenerator; /* unused parameter */
@@ -489,6 +491,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     PaInt32 temp;
 
     while( count-- )
@@ -525,6 +529,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     PaInt32 temp;
 
     (void) ditherGenerator; /* unused parameter */
@@ -560,6 +566,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     PaInt32 temp;
 
     while( count-- )
@@ -707,6 +715,8 @@
 {
     float *src = (float*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -728,6 +738,8 @@
 {
     float *src = (float*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void)dest;
+    (void)src;
 
     while( count-- )
     {
@@ -751,6 +763,8 @@
 {
     float *src = (float*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -773,6 +787,8 @@
 {
     float *src = (float*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -798,6 +814,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -819,6 +837,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
 
     while( count-- )
     {
@@ -842,6 +862,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -864,6 +886,8 @@
 {
     float *src = (float*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -909,6 +933,8 @@
 {
     PaInt32 *src    = (PaInt32*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void) ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -995,6 +1021,8 @@
 {
     PaInt32 *src = (PaInt32*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1015,6 +1043,8 @@
 {
     PaInt32 *src = (PaInt32*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     PaInt32 dither;
 
     while( count-- )
@@ -1037,6 +1067,8 @@
 {
     PaInt32 *src = (PaInt32*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1057,6 +1089,8 @@
 {
     PaInt32 *src = (PaInt32*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1275,6 +1309,8 @@
 {
     unsigned char *src = (unsigned char*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
 
     (void) ditherGenerator; /* unused parameter */
 
@@ -1366,6 +1402,8 @@
 {
     PaInt16 *src   = (PaInt16*) sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     PaInt16 temp;
 
     (void) ditherGenerator; /* unused parameter */
@@ -1398,6 +1436,8 @@
 {
     PaInt16 *src = (PaInt16*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1418,6 +1458,8 @@
 {
     PaInt16 *src = (PaInt16*)sourceBuffer;
     signed char *dest =  (signed char*)destinationBuffer;
+    (void)src;
+    (void)dest;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1438,6 +1480,8 @@
 {
     PaInt16 *src = (PaInt16*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1458,6 +1502,8 @@
 {
     PaInt16 *src = (PaInt16*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void)dest;
+    (void)src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1519,6 +1565,8 @@
 {
     signed char *src = (signed char*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1568,6 +1616,8 @@
 {
     signed char *src = (signed char*)sourceBuffer;
     unsigned char *dest =  (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void)ditherGenerator; /* unused parameter */
 
     while( count-- )
@@ -1629,6 +1679,8 @@
 {
     unsigned char *src  = (unsigned char*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
     (void) ditherGenerator; /* unused parameters */
 
     while( count-- )
@@ -1698,6 +1750,8 @@
 {
     unsigned char *src = (unsigned char*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
 
     (void) ditherGenerator; /* unused parameter */
 
@@ -1740,6 +1794,8 @@
 {
     unsigned char *src = (unsigned char*)sourceBuffer;
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
+    (void) src;
 
     (void) ditherGenerator; /* unused parameter */
 
@@ -1897,6 +1953,7 @@
         unsigned int count )
 {
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
 
     while( count-- )
     {
@@ -1912,6 +1969,7 @@
         unsigned int count )
 {
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
 
     while( count-- )
     {
@@ -1942,6 +2000,7 @@
         unsigned int count )
 {
     unsigned char *dest = (unsigned char*)destinationBuffer;
+    (void) dest;
 
     while( count-- )
     {
